public/configurations/visualizations/vss-domain-events-detail.json

```diff
@@ -9,7 +9,7 @@
         "columns": [
             { "column": "timestamp", "label": "Timestamp", "timeFormat": "%b %d, %y %X"},
             { "column": "value", "label": "Value" },
-            { "column": "sourceip", "label": "Source IP" },
+            { "column": "nuage_metadata.vportId", "label": "vportId IP", "totalCharacters": 8, "tooltip" : true },
```

Comment:
This is an intended change getting overwritten. Please revert this change. vportId is used somewhere else as well and you need to make the change there.